elk/run.py

```diff
@@ -31,6 +32,18 @@
 )
 
 
+def fetch_git_hash() -> str:
+    try:
+        return (
+            subprocess.check_output(["git", "rev-parse", "HEAD"])
```

Comment:
Doesn't this depend on the CWD being the git repo or something? If we want to add this feature we should use `__file__` or something similar to figure out where to run `git`

Reply:
Yeah, good catch, thanks. Added a test and made the function work even if run in not-the-git-repo.